gr-data.csv

```diff
@@ -473,6 +473,8 @@ https://github.com/sogis/gretl,9f547d3440f73c3d58a949c11f1c59cde9e68758,gretl,ch
 https://github.com/sogis/gretl,9f547d3440f73c3d58a949c11f1c59cde9e68758,gretl,ch.so.agi.gretl.steps.PublisherStepFile2LocalTest.file_regionsList,ID-HtF,,,
 https://github.com/sogis/gretl,9f547d3440f73c3d58a949c11f1c59cde9e68758,gretl,ch.so.agi.gretl.steps.PublisherStepFile2LocalTest.file_regionsRegEx,ID-HtF,,,
 https://github.com/sogis/gretl,9f547d3440f73c3d58a949c11f1c59cde9e68758,gretl,ch.so.agi.gretl.steps.PublisherStepFile2LocalTest.file_regionsUpdate,ID-HtF,,,
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.requestMappingIsInstrumented,NIO,,,
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.metrics.test.MetricsIntegrationTests.restTemplateIsInstrumented,NIO,,,
```

Comment:
Are these `NIO` when run in the same JVM or even in different JVMs?

Reply:
I have tested them in the same JVM and they are `NIO`.